### ISSUE TITLE
bitly ebay short link don't work

### ISSUE THREAD

**Issue by vrenda16** [lmammino/tall#19]
Hi, 

I have a issue when i try to unshort a bitly ebay link (https://ebay.to/xxxxx). The method return www.ebay.com url only. Someone have the same issue?

**Comment by lmammino** [comment]
Hello @vrenda16, do you have an actual valid link that i can play with to try to debug this issue? thanks :)

**Comment by mifi** [comment]
I have a similar issue:

```
require('tall').tall('http://bit.ly/fkWS88').then(console.log)
https://news.ycombinator.com/item
```

Should be `http://news.ycombinator.com/item?id=2025354` (query string missing from tall):

```
curl -v http://bit.ly/fkWS88
*   Trying 67.199.248.11...
* TCP_NODELAY set
* Connected to bit.ly (67.199.248.11) port 80 (#0)
> GET /fkWS88 HTTP/1.1
> Host: bit.ly
> User-Agent: curl/7.64.1
> Accept: */*
> 
< HTTP/1.1 301 Moved Permanently
< Server: nginx
< Date: Fri, 02 Apr 2021 15:18:37 GMT
< Content-Type: text/html; charset=utf-8
< Content-Length: 130
< Cache-Control: private, max-age=90
< Location: http://news.ycombinator.com/item?id=2025354
< Set-Cookie: _bit=l32fiB-81903003860e6a0962-00F; Domain=bit.ly; Expires=Wed, 29 Sep 2021 15:18:37 GMT
< Via: 1.1 google
< 
<html>
<head><title>Bitly</title></head>
<body><a href="http://news.ycombinator.com/item?id=2025354">moved here</a></body>
* Connection #0 to host bit.ly left intact
</html>
```

I think the key to the issue is the `http` instead of `https`

**Comment by lmammino** [comment]
Hello @mifi, thanks for reporting this. Which version of `tall` are you using?

**Comment by mifi** [comment]
Hi! Tested with 4.0.0, same issue

**Comment by mifi** [comment]
awesome!

**Comment by lmammino** [comment]
This should be fixed in 4.0.1 (just released) :)
